src/scss/_dropdown.scss

```diff
@@ -17,6 +17,7 @@
 }
 
 .select2-results {
```

Comment:
Why adding to .select2-results?
Actually the .select2-results__options is the scrolled element (max-height: 200px; overflow-y: auto;)